src/BuildScriptGenerator/Helpers/StaticSiteGeneratorHelper.cs

```diff
@@ -32,6 +33,16 @@ public static bool IsStaticSite(ISourceRepo sourceRepo)
         /// <returns>True if the app is a Hugo app, false otherwise.</returns>
         public static bool IsHugoApp(ISourceRepo sourceRepo)
         {
+            // Check for Hugo environment variables
+            var environmentVariables = Environment.GetEnvironmentVariables();
+            foreach (var key in environmentVariables?.Keys)
+            {
+                if (key.ToString().StartsWith("HUGO"))
+                {
+                    return true;
+                }
+            }
+
             // Hugo configuration variables: https://gohugo.io/getting-started/configuration/#all-configuration-settings
             var hugoVariables = new[] { "baseURL", "title", "languageCode", "theme" };
```

Comment:
Could you add few more variables at the beginning of this array which look more specific to Hugo maybe? (languageCode seems good but others i am not sure)...just want to avoid hitting false positives when probing.

Reply:
Added a few more configuration variables to this list.